tools/ttmlir-opt/CMakeLists.txt

```diff
@@ -26,13 +38,36 @@ VhloTypes)
 
 get_property(dialect_libs GLOBAL PROPERTY MLIR_DIALECT_LIBS)
 get_property(conversion_libs GLOBAL PROPERTY MLIR_CONVERSION_LIBS)
-set(LIBS ${stablehlo_libs} ${dialect_libs} ${conversion_libs} MLIROptLib MLIRTargetCpp TTMLIRStatic)
+set(LIBS TTMLIRStableHLOToTTIR TTMLIRCAPI ${stablehlo_libs} ${dialect_libs} ${conversion_libs} MLIROptLib MLIRTargetCpp 
```

Comment:
If stablehlo is disabled does this still work? It looks like it unconditionally adds `${stablehlo_libs}`